helm/formatting_guidelines.md

```diff
@@ -0,0 +1,59 @@
+# Helm Chart Formatting Guidelines
+
+## Templating
+
+When using helm templating calls we include double spaces inside the double brackets
+like this:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-configmap
+data:
+  myvalue: "Hello World"
+```
+
+In comparison to this:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  # No spaces makes it harder to see the templating call
+  name: {{.Release.Name}}-configmap
+data:
+  myvalue: "Hello World"
+```
+
+## Indentation
+
+The indentation of our helm chart yaml files should be 2 spaces.
+Moreover we do not indent yaml lists, hyphens of the list members act as indentation.
```

Comment:
According to helm docs we should indent lists: https://docs.helm.sh/developing_charts/#managing-dependencies-with-requirements-yaml

(That's also my personal preference - that's why I mention :P)

Reply:
Yes saw that helm style is different. I personally don't care which style we use, as long as its consistent, which is currently not the case ;)
Just saw that not-intending was mostly prevelent in our main operators repos (e.g. aws, kvm, azure deployments, unfortunatly not rbac, psp) and it seems to be the case most charts upstream https://github.com/kubernetes/charts/tree/master/stable and in kubernetes docs, e.g. https://kubernetes.io/docs/concepts/workloads/controllers/deployment/